modules/apps/object/object-service/src/main/java/com/liferay/object/internal/notification/term/evaluator/util/ObjectDefinitionNotificationTermEvaluatorUtil.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.internal.notification.term.evaluator.util;
+
+import com.liferay.list.type.model.ListTypeEntry;
+import com.liferay.list.type.service.ListTypeEntryLocalServiceUtil;
+import com.liferay.object.constants.ObjectFieldConstants;
+import com.liferay.object.model.ObjectField;
+import com.liferay.petra.function.transform.TransformUtil;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.sql.Timestamp;
+
+import java.text.SimpleDateFormat;
+
+import java.util.Date;
+
+/**
+ * @author Carolina Barbosa
+ */
+public class ObjectDefinitionNotificationTermEvaluatorUtil {
+
+	public static Object getObjectFieldTermValue(
+		ObjectField objectField, Object objectFieldValue) {
```

Comment:
Since the second parameter type is Object, I think it is a bit strange to be called "**objectField**Value."
I would call it `Object value`.

Analyzing the usages of this method, maybe it is okay to use a generic name for the second parameter :thinking: 
```
ObjectDefinitionNotificationTermEvaluatorUtil.getObjectFieldTermValue(
		objectField,
		termValues.get(objectField.getName())
ObjectDefinitionNotificationTermEvaluatorUtil.getObjectFieldTermValue(
		relatedObjectField,
		values.get(relatedObjectField.getName())
```

Reply:
Done